ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -45,3 +45,26 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+@given(
```

Comment:
Linting issues in this test function, you can see the exact errors in the workflow

Reply:
How to pass all the tests? I don't find the reason of this error.
should I implement the same function for tourch and numpy ?

Reply:
Done finishing lint issues.
for the torch and numpy error it is not in this task or you want me to add it ?
and for the docstrings you put in the docs that I don't have to implement it, is it right ?

Reply:
Yep, just the lint issue, I'll get this merged now